yb-voyager/cmd/import.go

```diff
@@ -496,12 +498,17 @@ func validateOnPrimaryKeyConflictFlag() error {
 	}
 
 	// ensure that OnPrimaryKeyConflictAction is not changed in case of resumption
-	if !startClean {
+	/*
+		Most of our cmd validations(including this) run in PreRun phase
+		which can happen before the migration status record or the metaDB is created.
+		For example, in case of import data file command
+	*/
+	if !bool(startClean) && metaDBIsCreated(exportDir) {
```

Comment:
Hmm, slightly confused. 
We call initMetaDB in the root command's PersistentPreRun. Isn't that run before the pre-run of the specific command?